gptme/prompts.py

```diff
@@ -143,7 +145,7 @@ def prompt_user() -> Generator[Message, None, None]:
     """
     config_prompt = get_config().prompt
     about_user = config_prompt.get(
-        "about_user", "You are interacting with a human programmer."
+        "about_user", f"You are interacting with a human programmer. I am running a {get_system_info()} system."
```

Comment:
I don't think we want to add it to the user prompt like this.

I'd like the config to be platform-portable (I copy mine between machines), and this wouldn't work for users with an old config.

`prompt_user` is also a bad way to put it, since it's not included in interactive mode. 

Maybe add a `prompt_systeminfo`?